File: backend/dotnet-services/UserRoleService/Cashier.Shared/DTOS/AppUser/AppUserDto.cs
namespace Cashier.Shared.DTOS.AppUser
{
    public class AppUserDto
    {
        public int Id { get; set; }
        public string Email { get; set; } = string.Empty;
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;
        public string PhoneBook { get; set; } = string.Empty;
        public List<string> Roles { get; set; } = new List<string>();
        public bool IsSuspended { get; set; }  
    }
}

File: backend/springboot-services/tenant-service/src/main/java/com/market_os/tenant_service/dto/AppUserDto.java
package com.market_os.tenant_service.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AppUserDto {
    
    private Integer id;
    
    private String firstname;
    
    private String lastname;
    
    private String email;
    
    @JsonProperty("is_suspended")
    private Boolean isSuspended;
} 